### PR TITLE
tests.pl: add --skipgeo option

### PR DESCRIPTION
**Pull request by matoro**
This allows capture tests to be run offline.  It assumes that the usual
Geo files are already present in the directory.  Needed for packaging
that doesn't allow network access at build/test time.

## License
I confirm that this contribution is made under an Apache 2.0 license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: tests/tests.pl [hunk 1]
```diff
@@ -468,6 +474,7 @@ sub doViewer {
     print "  --elasticsearch <url> Set elasticsearch URL\n";
     print "  --debug               Turn on debuggin\n";
     print "  --valgrind            Use valgrind on capture\n";
+    print "  --skipgeo             Skip downloading Geo files, assume already present (useful for offline)\n";
```
Comment (awick) [inline tests/tests.pl:477]: I'm confused why this change is needed, since if the files already exist it won't try and download them anyway, and the help says to assume they are present.

**Review by awick — CHANGES_REQUESTED** [review]

**Comment by matoro** [comment]
Oops, you're right - not sure why I didn't catch that this check already exists!  My bad.
